src/test/ui/consts/miri_unleashed/mutating_global.rs

```diff
@@ -1,4 +1,3 @@
-// compile-flags: -Zunleash-the-miri-inside-of-you
 #![allow(const_err)]
 
 // Make sure we cannot mutate globals.
```

Comment:
This test should now be moved to a different directory.

Reply:
Oh wait... no, this test is fine.

Reply:
This is very similar to https://github.com/rust-lang/rust/pull/71748#discussion_r418527661 -- the point of the test is to make sure the dynamic checks can catch this error no matter what the static checks are doing. But after removing the flag we don't know any more if that is the case.

Reply:
Yea this is already hitting dynamic checks because our static checks can only detect mutations of statics opportunistically. While we could have a static check right now, with mutable references on the horizon that check would just stop working again.